doc.go

```diff
@@ -27,12 +27,24 @@
 //
 // Particularly interesting subpackages include:
 //
-//   - impl/* -- various Node + NodeBuilder implementations
-//   - encoding/* -- functions for serializing and deserializing Nodes
-//   - linking/* -- various Link + LinkBuilder implementation
+//   - node/* -- various Node + NodeBuilder implementations
+//   - node/basic -- the first Node implementation you should try
+//   - codec/* -- functions for serializing and deserializing Nodes
+//   - linking/* -- various Link + LinkBuilder implementations
 //   - traversal -- functions for walking Node graphs (including
 //        automatic link loading) and visiting
-//   - typed -- Node implementations with constraints
-//   - fluent -- Node interfaces with streamlined error handling
+//   - must -- helpful functions for streamlining error handling
+//   - fluent -- alternative Node interfaces that flip errors to panics
+//   - schema -- interfaces for working with IPLD Schemas and Nodes
+//        which use Schema types and constraints
+//
+// Note that since packages in this interface are the core of the library,
```

Comment:
"interfaces in this package"? "interfaces in these packages"?

Reply:
woof